ui/packages/tidb-dashboard-lib/src/apps/SystemReport/context/index.ts

```diff
@@ -29,8 +29,13 @@ export interface ISystemReportDataSource {
   ): AxiosPromise<DiagnoseReport>
 }
 
+export interface ISystemReportEvent {
+  getFullReportLink(reportID: string): string
+}
```

Comment:
Why `getFullReportLink` is an event? Since no one subscribed it out of the app.

Reply:
Yep, it's a bad name, do you have any suggestion for a better name?